indexing-service/src/test/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorStateTest.java

```diff
@@ -1326,6 +1339,9 @@ protected Map<String, Long> getPartitionTimeLag()
     @Override
     protected void emitLag()
     {
+      if ((metricFlag & LAG) == 0) {
```

Comment:
This is a nice approach but we would only be able to track 8 metrics at most. Wouldn't it be simpler to just do `metricFlag != LAG`?

Reply:
did it like this for scenarios when we want to test the emission of two different metrics. I don't have such a test case now so I am happy to change the implementation.

Reply:
No, that makes sense. As it is just inside this test, this is fine too. We can change it later if we need to.